### DISCUSSION TITLE
FFMPEG: moov atom not found

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#766] [Help!] [answered]
Hello,

It looks like this might be a bug. The list of videos in my project aren't showing up. Is it possible that other people have seen this, and is it recoverable? I can't use the .slp file, add videos, etc., even though I can flip from video to video and from frame to frame, the "Videos" window is empty. Is it possible the file is corrupt? I have an older version of this but I've added 1.5k labelled frames since then. Typically I iterate the .slp file since I saw this happen roughly a year ago, and backing up to previous versions of the .slp before moving forward has helped. It seems like the only common part is that both times, I was trying to merge this project into another.

![image](https://user-images.githubusercontent.com/69451171/168523424-bc03ba20-d580-4313-9450-f091244328f6.png)

File can be found here: https://drive.google.com/file/d/1j05LhwzqOSM3x7OA7WVqHAuKIP8WnAzY/view?usp=sharing

Thanks for your time,
Stefan

**Comment by talmo** [comment]
Hi @olinesn,

Do you see any errors in the terminal?

Talmo

**Comment by olinesn** [comment]
Interesting - "moov atom not found"

![image](https://user-images.githubusercontent.com/69451171/168658224-9f91e4b2-d9f7-4c27-b857-771c335df4de.png)

So I suppose this could be related to me reinstalling Google Drive on my machine? It's strange since it can clearly see the data.

**Comment by roomrys** [comment]
Hi @olinesn,

Did messing with your Google Drive installation help at all? Or are you still struggling with the getting the videos to display?

Thanks,
Liezl

**Comment by talmo** [comment, marked as answer]
It could be the Google Drive thing which leads to the file "appearing" to be corrupted or, [as per this Reddit thread](https://old.reddit.com/r/ffmpeg/comments/ih0hf7/ffmpeg_moov_atom_not_found/g2x2vox/), it might have been an issue with the acquisition or re-encoding terminating early.

What are you using for acquisition again @olinesn?

**Comment by olinesn** [comment]
Ok it looks like it was similar to the issure you linked. There's something strange that was happening with a single .mp4 file when it was syncing. The file size was correct but the video looked corrupted, even though Google Drive didn't identify the problem. I downloaded the file manually from the web interface and replaced it, and things are working now. Thanks for the tip!

**Comment by roomrys** [comment]
Glad that it is working @olinesn. Converting this to a discussion in case someone else comes across the same issue.

**Comment by ellielo** [comment]
it’s possible that something went wrong during the merging process. You might want to check if the project file is still referencing the correct video paths or if any recent changes caused the issue. Since file corruption is a possibility, you could try recovery solutions like the ones mentioned [https://www.handyrecovery.com/moov-atom-not-found/](https://www.handyrecovery.com/moov-atom-not-found/) which discuss fixing missing metadata in media files. If you haven't already, restoring from a previous backup and reattempting the merge step-by-step might help pinpoint the cause.
